examples/keras_io/vision/image_classifier.py

```diff
@@ -0,0 +1,692 @@
+    value_range=(0, 255),
+    show=True,
+)
+
+"""Looks great; but we're not done yet!
+What if an image is missing one critical feature of a class?  For example,
+what if a leaf is blocking the view of a cat's ear, but our classifier
+learned to classify cats simply by observing their ears?
+
+One easy approach to tackling this is to use `RandomCutout`, which randomly
+strips out a sub-section of the image:
+"""
+
+random_cutout = keras_cv.layers.RandomCutout(
+    width_factor=0.4, height_factor=0.4
+)
+keras_cv.visualization.plot_image_gallery(
+    random_cutout(image_batch),
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+
+"""While this tackles the problem reasonably well, it can cause the classifier
+to develop responses to borders between features and black pixel areas caused
+by the cutout.
+
+[`CutMix`](https://arxiv.org/abs/1905.04899) solves the same issue by using
+a more complex (and more effective) technique.
+Instead of replacing the cut-out areas with black pixels, `CutMix` replaces
+these regions with regions of other images sampled from within your training
+set!
+Following this replacement, the image's classification label is updated to be
+a blend of the original and mixed image's class label.
+
+What does this look like in practice?  Let's check it out:
+"""
+
+cut_mix = keras_cv.layers.CutMix()
+# CutMix needs to modify both images and labels
+inputs = {"images": image_batch, "labels": label_batch}
+
+keras_cv.visualization.plot_image_gallery(
+    cut_mix(inputs)["images"],
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+
+"""Let's hold off from adding it to our augmenter for a minute - more on that
+soon!
+
+Next, let's look into `MixUp()`.
+Unfortunately, while `MixUp()` has been empirically shown to *substantially*
+improve both the robustness and the generalization of the trained model,
+it is not well-understood why such improvement occurs... but
+a little alchemy never hurt anyone!
+
+`MixUp()` works by sampling two images from a batch, then proceeding to
+literally blend together their pixel intensities as well as their
+classification labels.
+
+Let's see it in action:
+"""
+
+mix_up = keras_cv.layers.MixUp()
+# MixUp needs to modify both images and labels
+inputs = {"images": image_batch, "labels": label_batch}
+
+keras_cv.visualization.plot_image_gallery(
+    mix_up(inputs)["images"],
+    rows=3,
+    cols=3,
+    value_range=(0, 255),
+    show=True,
+)
+
+"""If you look closely, you'll see that the images have been blended together.
+
+Instead of applying `CutMix()` and `MixUp()` to every image, we instead pick
+one or the other to apply to each batch.
+This can be expressed using `keras_cv.layers.RandomChoice()`
+"""
+
+cut_mix_or_mix_up = keras_cv.layers.RandomChoice(
+    [cut_mix, mix_up], batchwise=True
+)
+augmenters += [cut_mix_or_mix_up]
+
+"""Now let's apply our final augmenter to the training data:"""
+
+augmenter = tf.keras.Sequential(augmenters)
```

Comment:
There should be no tf.keras at all

Reply:
Let's use the `Augmenter` which we're adding back to KerasCV in https://github.com/keras-team/keras-cv/pull/1978